src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBlockControl/SimpleText_MaxLines_Two_With_Wrap.xaml.cs

```diff
@@ -9,6 +10,21 @@ public sealed partial class SimpleText_MaxLines_Two_With_Wrap : UserControl
 		public SimpleText_MaxLines_Two_With_Wrap()
 		{
 			this.InitializeComponent();
+
+
+#if __WASM__
+
+			var initialHits = FeatureConfiguration.TextBlock.CacheHits;
```

Comment:
That should go on the cache itself.

Reply:
I created a `UnoMetrics` class because I don't was to put that in the `Windows.UI.Xaml` namespaces. What do you think, @jeromelaban?